pkg/ipfs/node.go

```diff
@@ -339,6 +339,9 @@ var localOnlyProfile = config.Profile{
 		c.Swarm.RelayClient.Enabled = config.False
 		c.Swarm.RelayService.Enabled = config.False
 		c.Swarm.Transports.Network.Relay = config.False
+		c.Swarm.DisableNatPortMap = true
+		c.Addresses.Gateway = []string{}
+		c.Discovery.MDNS.Enabled = false
```

Comment:
mdns discovery looks potentially useful for bootstrapping in private networks in future, will make a note.